src/Entity/Board.php

```diff
@@ -119,12 +120,14 @@ public function getPucks()
             return $vertex->getValue() && array_key_exists(Board::COLOR_KEY, $vertex->getValue());
         });
     }
-    public function getPucksIdsByColor($color)
```

Comment:
Since PHP 7.0, you can type the params and the return of the function (https://www.php.net/manual/en/migration70.new-features.php#migration70.new-features.return-type-declarations) and other feature like null coalescing operator.

On PHP 8.0, you have some interest new feature too https://www.php.net/releases/8.0/en.php (like match expression).

The would look more like recent php code. See with @alexisjanvier if it's blocking for the project